src/composables/trade/__mocks__/mockSorSwapInfo.ts

```diff
@@ -0,0 +1,64 @@
+export default function mockSorSwapInfo() {
```

Comment:
Using this kind of hardcoded untyped mocks is painful cause:
- Tests will get broken whenever the SDK team changes the `SwapInfo` type. 
- The error will wont't be easy to spot 
-  It will require manually fixing this file.  

The alternative is simple: using jest/vitest-mock-extended to automatically match the type.

That is, replacing the whole content of this file with: 

```ts
import { SwapInfo } from '@balancer-labs/sdk';
import { mock } from 'jest-mock-extended';

export default function mockSorSwapInfo() {
  return mock<SwapInfo>();
}
```

Reply:
Same goes for `mockSorOutput.json` and `mockTokensOutput.json`, but let's leave that change for incoming PRs where I will suggest a simpler alternative.

Reply:
Only thing I'd suggest is if we can trim down those mocks, they are HUGE.

Reply:
@agualis Sure, thanks. For this test to work, there needs to be a very specific combination of pool, tokens and amounts - will this be preserved?

Reply:
Check [this](https://github.com/balancer-labs/frontend-v2/pull/2414/commits/b5665bc0cd7f3d90b4f7f84c13e5ef63ecc2bbed) out. It is far from being my preferred solution but it avoids the huge jsons. 

I have much more ideas to improve this farther (in incoming discussions/PRs).

Please check if the key parts are preserved or I can add them otherwise.

Reply:
Agree and, as I said, I would improve this farther with more time. Let's have a chat to double check that it tests the same logic that it was testing before.

Reply:
I just pushed a new version that: 
1. Covers useJoinExit as it used to
2. Uses much smaller mocks and moved from json to TS so that tests are less fragile

Again, far from being the best solution but it should work for now.